Documentation/CodeSnippets/FlexForms/Examples/PluginHaikuListRegistration.rst.txt

```diff
@@ -19,21 +20,30 @@
 
     defined('TYPO3') or die();
 
+    $pluginSignature = 'examples_haiku_list';
```

Comment:
Plugin Signature is extkey without underscore + `_` + plugin name without underscore. So, it should be `examples_haikulist` or `exampleshaiku_list`.

Reply:
Actually in non-Extbase plugins this is just an arbitary string tha you can define yourself?

Reply:
This needs to be changed in the example extension and will automatically be updated here. If you want to make a PR, go ahead!